documentation/modules/rn-2.2.adoc

```diff
@@ -9,21 +9,29 @@ You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {vir
 
 The release notes describe new features and enhancements, known issues, and technical changes.
 
-// [id="technical-changes-22_{context}"]
-// == Technical changes
-//
-// .TBD
-//
-// TBD
-//
-// [id="new-features-and-enhancements-22_{context}"]
-// == New features and enhancements
-//
-// This release adds the following features and improvements.
-//
-// .TBD
-//
-// TBD
+[id="technical-changes-22_{context}"]
+== Technical changes
+
+.Setting the precopy interval for warm migration
+
+You can set the interval between snapshots during the precopy stage of warm migration.
```

Comment:
@sbeskin-redhat @apinnick 
Can this sentence be changed to: 
"You can set the interval between snapshots **taken** during the precopy stage of warm migration"?

Reply:
Thanks. I think that according to the current sentence - the user can get the idea that the interval value can be changed during the warm migration run

Reply:
I see your point. Good catch.